docs/user/troubleshooting/first-troubleshooting-guide.md

```diff
@@ -0,0 +1,21 @@
+# Istio troubleshooting guide
+
+If you are experiencing issues with the cluster, you can follow the steps below to troubleshoot the problem before you contact us. This guide is designed to help you identify the root cause of the problem which might not necessarily be related to the Istio itself but rather to the misconfiguration of the Istio Custom Resources or other network related resources.
+
+## Try to go through the following steps:
+
+1. When an Istio problem is reported for a Kyma module, SRE should first check whether it also occurs for other modules. If the problem only occurs with a specific module, the team that owns that module should start investigating first before involving us.
```

Comment:
The list was created, before we discussed this with SRE. They made some improvements to their [issue template](https://raw.github.tools.sap/kyma/backlog/master/.github/ISSUE_TEMPLATE/internal-incident.md), so I think we can remove this. It's also more generic to processes of SRE and not Istio.
I personally would completely remove this step.